README.rst

```diff
@@ -121,9 +121,10 @@ The Renku project consists of several sub-repositories:
   Knowledge Graph services
 
 - `renku-notebooks <https://github.com/SwissDataScienceCenter/renku-notebooks>`_:
-  a lightweight service for handling interactive notebooks through JupyterHub
+  a lightweight service for handling interactive notebooks through Jupyter servers
+  and a k8s operator
```

Comment:
shouldn't we add Amalthea's repo here as well?

Reply:
Sure I will add it.